modules/nw-network-flows-object.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/tracking-network-flows.adoc
+
+[id="nw-network-flows-object_{context}"]
+= Network object configuration for tracking network flows
```

Comment:
fwiw I've adopted a table style for describing object fields: https://docs.openshift.com/container-platform/4.7/networking/cluster-network-operator.html#nw-operator-cr-cno-object_cluster-network-operator

Reply:
I'm submitting and update that gives that a try.  TY.